src/FileOnQ.Imaging.Raw/LibRaw/Parameters.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Runtime.InteropServices;
+
+namespace FileOnQ.Imaging.Raw
+{
+	public unsafe partial class LibRaw
+	{
+		[StructLayout(LayoutKind.Sequential, Pack = 1)]
+		public struct Parameters
```

Comment:
This file was copied from LibRaw and some names were adjusted to match C# naming conventions